js/views/onboardingModal.js

```diff
@@ -139,7 +132,7 @@ module.exports = baseModal.extend({
         self = this;
 
     this._accordianReady = true;
-    this.loadingOff();
+    this.loadingModal.close();
 
     this.initAccordion('.js-profileAccordion');
     
```

Comment:
Line 250 should be set to make the guidStillCreatingModal non-closable, otherwise if the guid takes a long time you can close that modal, which causes issues.

I think this would do it:
this.guidStillCreatingModal = new guidStillCreatingModal({
        dismissOnOverlayClick: false,
        dismissOnEscPress: false,
        showCloseButton: false
      });